src/overlay/OverlayManagerImpl.cpp

```diff
@@ -871,6 +871,12 @@ OverlayManagerImpl::isPreferred(Peer* peer) const
     return false;
 }
 
+bool
```

Comment:
adding comment here - I think we should add flow control information to the "peers" endpoint

Reply:
i suggest we do this in a subsequent pr

Reply:
ok let's open an issue for this then